src/amo/components/SearchContextCard/index.js

```diff
@@ -29,67 +29,121 @@ export class SearchContextCardBase extends React.Component {
 
   render() {
     const { count, filters, i18n, loading } = this.props;
-    const { query } = filters;
+    const { query, category } = filters;
     const { addonType } = filters;
 
     let searchText;
 
-    if (!loading && query && addonType) {
-      if (addonType === ADDON_TYPE_EXTENSION) {
-        searchText = i18n.sprintf(
-          i18n.ngettext(
-            '%(count)s extension found for "%(query)s"',
-            '%(count)s extensions found for "%(query)s"',
-            count,
-          ),
-          { count: i18n.formatNumber(count), query },
-        );
-      } else if (addonType === ADDON_TYPE_DICT) {
-        searchText = i18n.sprintf(
-          i18n.ngettext(
-            '%(count)s dictionary found for "%(query)s"',
-            '%(count)s dictionaries found for "%(query)s"',
-            count,
-          ),
-          { count: i18n.formatNumber(count), query },
-        );
-      } else if (addonType === ADDON_TYPE_LANG) {
-        searchText = i18n.sprintf(
-          i18n.ngettext(
-            '%(count)s language pack found for "%(query)s"',
-            '%(count)s language packs found for "%(query)s"',
-            count,
-          ),
-          { count: i18n.formatNumber(count), query },
-        );
-      } else if (addonType === ADDON_TYPE_THEME) {
-        searchText = i18n.sprintf(
-          i18n.ngettext(
-            '%(count)s theme found for "%(query)s"',
-            '%(count)s themes found for "%(query)s"',
-            count,
-          ),
-          { count: i18n.formatNumber(count), query },
-        );
-      } else {
-        searchText = i18n.sprintf(
-          i18n.ngettext(
-            '%(count)s result found for "%(query)s"',
-            '%(count)s results found for "%(query)s"',
-            count,
-          ),
-          { count: i18n.formatNumber(count), query },
-        );
+    if (!loading) {
+      switch (addonType) {
+        case ADDON_TYPE_EXTENSION:
+          if (category) {
+            const categoryName = category.replace('-', ' ');
```

Comment:
We cannot do this unfortunately. We need translated strings, which means a lot of new strings and/or new API calls to load the categories.

Reply:
thanks for the quick feedback. I'll revisit this one...

Reply:
actually, quick question on this...

doesn't sprintf in combination with i18n.ngettext translate this? I am not clear why this won't work :/

Reply:
That's because the category name needs to be translated too